README.md

```diff
@@ -749,6 +749,17 @@ const token = await stripe.createTokenWithBankAccount(params)
 // api.sendTokenToBackend(token)
 ```
 
+**NOTE** The Routing number is a 7 digit number which is a combination of `Bank code` and `Branch code`. Bank code is 4 digit number and Branch code is 3 digit number. By combine that two respected number we can make routing number. Parameter to pass in createTokenWithBankAccount method
+
+```
+const params = {
+  accountNumber: this.state.accountNo,
+  countryCode: 'sg',
+  currency: 'sgd',
+  routingNumber: `${this.state.bankCode}${this.state.branchCode}`
+}
+```
+
```

Comment:
There is no example Component to use next fields:
```js
this.state.accountNo
this.state.bankCode
this.state.branchCode
```

Add links for the `Bank Code` and `Branch Code`. I don't know what it means.

Reply:
https://en.wikipedia.org/wiki/Bank_code 
https://www.quora.com/What-is-a-branch-code-of-a-bank i.e. first 4 digits of account number.
I'm gonna update docs soon.